src/features/user/MFV2/ContributionsMap/Markers/CustomMarkers.tsx

```diff
@@ -0,0 +1,125 @@
+import { Marker } from 'react-map-gl-v7';
+import RegisteredTreeIcon from '../../../../../../public/assets/images/icons/myForestV2Icons/RegisteredTreeIcon';
+import NaturalRegeneration from '../../../../../../public/assets/images/icons/myForestV2Icons/NaturalRegeneration';
+import Mangroves from '../../../../../../public/assets/images/icons/myForestV2Icons/Mangroves';
+import ManagedRegeneration from '../../../../../../public/assets/images/icons/myForestV2Icons/ManagedRegeneration';
+import Agroforestry from '../../../../../../public/assets/images/icons/myForestV2Icons/Agroforestry';
+import UrbanRestoration from '../../../../../../public/assets/images/icons/myForestV2Icons/UrbanRestoration';
+import Conservation from '../../../../../../public/assets/images/icons/myForestV2Icons/Conservation';
+import TreePlanting from '../../../../../../public/assets/images/icons/myForestV2Icons/TreePlanting';
+import OtherPlanting from '../../../../../../public/assets/images/icons/myForestV2Icons/OtherPlanting';
+import { contributionLocation } from '../../../../../utils/myForestV2Utils';
+import themeProperties from '../../../../../theme/themeProperties';
+
+interface ProjectTypeIconProps {
+  projectType: string;
+  classification: string;
+}
+
+const ProjectTypeIcon = ({
+  projectType,
+  classification,
+}: ProjectTypeIconProps) => {
+  const chooseColorForMarkerIcon = (classification: string) => {
+    switch (classification) {
+      case 'treePlantation':
+        return `${themeProperties.primaryDarkColorX}`;
+      case 'conservation':
+        return `${themeProperties.mediumBlue}`;
+      case 'restoration':
+        return `${themeProperties.electricPurple}`;
+      default:
+        return null;
+    }
+  };
```

Comment:
1. Should null be a possible return value? That indicates that we haven't considered a valid value.
2. `classification` could be typed more specifically, so that we are warned while using an invalid value